packages/modules/src/modules/banners/investigationsMoment/components/InvestigationsMomentBannerCtas.tsx

```diff
@@ -191,15 +201,85 @@ function ButtonWithPaymentIcons({ button }: ButtonWithPaymentIconProps) {
     return (
         <div css={buttonWithPaymentIconStyles.container}>
             {button}
-
-            <div css={buttonWithPaymentIconStyles.paymentIconsContainer}>
-                <img
-                    width={497}
-                    height={88}
-                    src="https://media.guim.co.uk/40745a456b9da26eccca15a615dd0e406839ceb6/0_0_1549_274/500.png"
-                    alt="Accepted payment methods: Visa, Mastercard, American Express and PayPal"
-                />
+            <div>
+                <PaymentCards />
             </div>
         </div>
     );
 }
+
+// ---- Helper Components ---- //
+
+function PaymentCards() {
```

Comment:
I reckon this could be it's own component

Reply:
Yes, makes sense, have done a common/PaymentCards.tsx which allows cssOverides (in case needs to be styled). Hopefully have done it the correct way.